src/components/display-dices.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const DispalyDices = (props) => (
+    <div style={props.dice1 === 1 || props.dice2 ===1 ? {display: 'none'} : {display: 'block'}}>
```

Comment:
dice1, dice2 is not good naming method practice -> firstDice, secondDice.
You should user props destruct: { firstDice, secondDice } = props;
Styles should be moved to separated file. You should operate on the classNames.